packages/workspace/src/command-line/nx-commands.ts

```diff
@@ -333,39 +335,41 @@ function withRunManyOptions(yargs: yargs.Argv): yargs.Argv {
       return true;
     })
     .options('runner', {
-      describe: 'Override the tasks runner in `nx.json`',
+      describe: 'Override the tasks runner in `nx.json`.',
       type: 'string',
     })
     .options('skip-nx-cache', {
       describe:
-        'Rerun the tasks even when the results are available in the cache',
+        'Rerun the tasks even when the results are available in the cache.',
       type: 'boolean',
       default: false,
     })
     .options('configuration', {
       describe:
-        'This is the configuration to use when performing tasks on projects',
+        'This is the configuration to use when performing tasks on projects.',
       type: 'string',
     })
-    .options('with-deps', {
+    .option('with-deps', {
       describe:
-        'Include dependencies of specified projects when computing what to run',
+        'Include dependencies of specified projects when computing what to run.',
       type: 'boolean',
       default: false,
+      deprecated:
+        'Configure [target dependencies](/{{framework}}/core-concepts/configuration#target-dependencies) instead. It will be removed in v14.',
```

Comment:
This doesn't look good too good in the terminal:

```
  --with-deps      Include dependencies of specified projects when computing
                   what to run                    [deprecated: Configure [target
   dependencies](/{{framework}}/core-concepts/configuration#target-dependencies)
                 instead. It will be removed in v14.] [boolean] [default: false]
                 ```

Reply:
Good catch! I was only thinking in the generated docs, but I forgot to check the actual command help 🤦🏻